core/src/main/java/com/amplifyframework/util/UserAgent.java

```diff
@@ -20,7 +20,7 @@
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
 
-import com.amazonaws.amplify.core.BuildConfig;
```

Comment:
It might be worth taking this and the corresponding change above out of this PR and include it in a separate commit against main as a fix.

Reply:
Sounds good, I created a PR for these two changes: https://github.com/aws-amplify/amplify-android/pull/616